FirebaseInstallations.podspec

```diff
@@ -35,6 +35,7 @@ Pod::Spec.new do |s|
     s.framework = 'Security'
     s.dependency 'FirebaseCore', '~> 6.0'
     s.dependency 'PromisesObjC', '~> 1.2'
+    s.dependency 'GoogleUtilities/UserDefaults', '~> 6.0'
```

Comment:
Since it's a new pod, start with the current version - 6.2.  We don't have a good test strategy for earlier versions.